resources/type-system/inference.md

```diff
@@ -150,22 +153,27 @@ error if there is such an ordering.  Note that method override inference is
 independent of non-override inference, and hence can be completed prior to the
 rest of top level inference if desired.
 
+
 ##### Method override inference
 
-A method which is subject to override inference is missing one or more component
-types of its signature, and it overrides one or more declarations. Each missing
-type is filled in with the corresponding type from the overridden or implemented
-method.  If there are multiple overridden/implemented methods, and any two of
-them have non-equal types (declared or inferred) for a parameter position which
-is being inferred for the overriding method, it is an error.  If there is no
-corresponding parameter position in the overridden method to infer from and the
-signatures are compatible, it is treated as dynamic (e.g. overriding a one
-parameter method with a method that takes a second optional parameter).  Note:
-if there is no corresponding parameter position in the overridden method to
-infer from and the signatures are incompatible (e.g. overriding a one parameter
-method with a method that takes a second non-optional parameter), the inference
-result is not defined and tools are free to either emit an error, or to defer
-the error to override checking.
+A method `m` of a class `C` which is subject to override inference is
```

Comment:
Maybe "A method `m` of a class `C` is subject to override inference if it"

Reply:
Done.